### ISSUE TITLE
Icon role prop no longer defaults to `presentation`

### ISSUE THREAD

**Issue by MustafaHaddara** [chakra-ui/chakra-ui-docs#70]
### Subject

Icon role prop no longer defaults to `presentation`

### Description

seems like this prop stopped receiving `presentation` as a default, as per https://github.com/chakra-ui/chakra-ui/pull/1349 and https://github.com/chakra-ui/chakra-ui/issues/1197

however the documentation still lists `presentation` as the default: https://chakra-ui.com/docs/media-and-icons/icon#props

**Comment by lazarnikolov94** [comment]
Good catch @MustafaHaddara! Are you interested in opening a PR that fixes this?
